inc/ctrs_leak_check.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef CTRS_LEAK_CHECK_H
+#define CTRS_LEAK_CHECK_H
+
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
+#endif
+
+#include "testrunnerswitcher.h"
```

Comment:
>testrunnerswitcher [](start = 10, length = 18)

This file (ctrs_leak_check) should include vld.h

Reply:
LIkely under VLD_OPT_REPORT_TO_STDOUT

---
In reply to: [291298918](https://github.com/Azure/azure-c-testrunnerswitcher/pull/19#discussion_r291298918) [](ancestors = 291298918)

Reply:
I'll add it. Truth be told this is piggybacking the fact that vld.h was included. But maybe in the future we should have an explicit switch for this. But maybe in the future :-). Let's not do all the improvements in one day.

---
In reply to: [291299367](https://github.com/Azure/azure-c-testrunnerswitcher/pull/19#discussion_r291299367) [](ancestors = 291299367,291298918)